extension/intents/navigation/clickLogin.js

```diff
@@ -0,0 +1,31 @@
+/* globals communicate, log */
+
+this.followLink = (function() {
+  communicate.register("signIn", message => {
+    const { query } = message;
+    const regex = /log\s*in|sign\s*in/i;
+
+    for (const link of findButton()) {
+      const element = link.innerText;
+      return regex.test(element.innerText);
+    }
+    const matches = regex.match(query);
+    const found = matches[0];
```

Comment:
This still has the same problem as before, and I don't think it will work at all. Be sure to test this!

The `return` statement will terminate the function as soon as you reach it! That means everything below the `for` loop will be skipped.

Also the next statements getting matches and assigning `found` don't really make sense.

Reply:
Yeeeaaah. My bad I did not commit correctly